tests/src/test/scala/common/Wsk.scala

```diff
@@ -1003,11 +1003,12 @@ trait RunWskCmd extends Matchers {
             verbose: Boolean = false,
             env: Map[String, String] = Map("WSK_CONFIG_FILE" -> ""),
             workingDir: File = new File("."),
+            stdinFile: File = null,
```

Comment:
perhaps it's just me, but method signatures with a mix of `Option` parameters and named default parameters is a bit confusing.
https://stackoverflow.com/questions/4199393/are-options-and-named-default-arguments-like-oil-and-water-in-a-scala-api